cmd/feedback/feedback.go

```diff
@@ -299,26 +338,30 @@ func executeSurvey(ctx context.Context, clients *shared.ClientFactory, s SlackSu
 	if s.Info != nil {
 		s.Info(ctx, clients)
 	}
-	clients.IO.PrintTrace(ctx, s.Trace)
+	clients.IO.PrintTrace(ctx, s.Trace, s.Name)
```

Comment:
```suggestion
	clients.IO.PrintTrace(ctx, slacktrace.FeedbackMessage, s.Name)
```

🪓 Since the same trace argument is used it might be more clear to hardcode that here? No blocker at all though!

Reply:
I'm all for axing it! 🪓 

I've pulled this work out to PR #68, which:

* Removes the `Trace` field from the `Survey` struct
* Always uses `slacktrace.FeedbackMessage`
* Adds tests to check that the trace is printed